src/simmi/TestCase14/TestCase13NG.java

```diff
@@ -0,0 +1,84 @@
+/*TestCase 14: 19th Jun'2021 
+	Remove main method from all assignments and use TestNG annotations.
+ 
+ * program 1: count number of employee under each manager, verify count against properties file.
+output : 20209 -> 3
+         20205 -> 4*/
+
+package simmi.TestCase14;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Properties;
+import java.util.Set;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class TestCase13NG {
+	
+	HashMap<String, Integer> employeeUnderManager(WebDriver driver) {
```

Comment:
If you create driver at instance level so we not require to pass reference of driver in every method